discovery-provider/plugins/verified-uploads/src/handlers/users.js

```diff
@@ -0,0 +1,61 @@
+export default async ({ slack, dp_db, id_db }, { user_id }) => {
+  const result = await dp_db("users")
+    .select("handle")
+    .where("user_id", "=", user_id)
+    .orderBy("blocknumber", "desc")
+    .limit(2)
+    .catch(console.error);
+
+  const current = result[0];
```

Comment:
doubt it's probable but it is possible for the very first record to have everything in it. since we index block by block, we could have the first create and verify consolidate into the same row. in that case where old row is undefined, we can assume the user is verified